mysql-to-kv/function.yaml

```diff
@@ -0,0 +1,53 @@
+
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: "nuclio.io/v1beta1"
+kind: "Function"
+spec:
+  description: "A simple v3io-webapi usage for PutItem using MySQL public DB"
+  handler: "main:handler"
+  env:
+    - type: value
+      name: TABLE
+      value: table
+    - type: value
+      name: CONTAINER
+      value: bigdata
+    - type: value
+      name: IGZ_USER
+      value: iguazio
+    - type: value
+      name: IGZ_PWD
+      value: dIusluqnAoJl5wNV
+    - type: value
+      name: SQL_HOST
+      value: mysql-rfam-public.ebi.ac.uk
+    - type: value
+      name: SQL_PORT
+      value: "4497"
+    - type: value
+      name: SQL_USER
+      value: rfamro
+    - type: value
+      name: SQL_DB_NAME
+      value: Rfam
+  runtime: "python:3.6"
+  minReplicas: 1
+  maxReplicas: 1
+  build:
+    commands:
+      - 'pip install requests pymysql pandas'
+    baseImage: "python:3.6-jessie"
```

Comment:
I would opt for `python:3.6` - it's used more often and also distributed to dark sites